components/post_view/post/post.test.js

```diff
@@ -183,4 +184,58 @@ describe('Post', () => {
             });
         }
     });
+
+    test('should show the thread footer for root posts', () => {
```

Comment:
we should probably have a test to see that the thread footer doesn't appear in case of it being a comment or a single post with no comments